### ISSUE TITLE
7.82.1 breaks CiviCRM

### ISSUE THREAD

**Issue by sluc23** [omega8cc/drupal7x#7]
latest release 7.82.1 breaks CiviCRM integration due to JS error:
```
Uncaught TypeError: CRM.$ is not a function
```
was working fine in your previous release 7.80.1

**Comment by omega8cc** [comment]
Have you tried this with official 7.82 vanilla core? We haven't changed anything beyond the vanilla core patches/commits being added.

**Comment by sluc23** [comment]
yes, it works with vanilla D7.82

**Comment by reswild** [comment]
I had a look at this, and I could see that all my aggregated javascript went missing after migrating to 7.82.1. I got them to load again by reverting the changes that had been made to the drupal_pre_render_scripts() function. I've made a PR for you with this change, although you might want to review the whole javascript aggregation code. I'm not sure if I fully understand all the changes you have made to this compared to how it's done in the vanilla code.
